### DISCUSSION TITLE
Receivers

### DISCUSSION THREAD

**Discussion by MarkMontreal** [ExpressLRS/ExpressLRS#899] [Q&A]
I can use many receivers, for many drones,  with my one radio right?  So I just have to build or use the same elrs configurator rx firmware on my second and more drones?  Isn't it dangerous when two or more of those drones are turned on (unawares or forgot to turn off let's say) at same time then?

**Comment by avsaase** [comment]
You can use multiple receivers with the same binding phrase (and other settings) and they will all work with your TX module. You are right that if you power multiple receivers at the same time your TX controls all of them simultaneously. In 2.0 there will be the model match feature so you can link a receiver to a specific model on your OpenTX/EdgeTX radio to avoid this problem. Untill then you should make sure not to power multiple receivers at the same time.

Just FYI, you should join the discord if you have more questions. Lots of helpful people there. https://discord.gg/SWet2V39

**Comment by MR444** [comment]
Since we passed already 2.0, where can I find how to match my models to the receiver mode?

I have different receivers for different models (diversity, gemini, single...) but I do not want to change the transmitter mode in the LUA script each time I change the model.

**Reply by pkendall64** [reply]
https://www.expresslrs.org/software/model-config-match/

**Reply by MR444** [reply]
Okay so model match is the keyword I have to search for. 
After I read a bit, shouldn't be a model numer enough instead of model matching? Or do I need both?
Thank you!
